StoreKitUnitTests/PurchasesOrchestratorTests.swift

```diff
@@ -60,7 +60,27 @@ class PurchasesOrchestratorTests: StoreKitConfigTestCase {
         mockManageSubsHelper = MockManageSubscriptionsHelper(systemInfo: systemInfo,
                                                                        customerInfoManager: customerInfoManager,
                                                                        identityManager: identityManager)
-        mockBeginRefundRequestHelper = MockBeginRefundRequestHelper(systemInfo: systemInfo)
+        mockBeginRefundRequestHelper = MockBeginRefundRequestHelper(systemInfo: systemInfo,
+                                                                    customerInfoManager: customerInfoManager,
+                                                                    identityManager: identityManager)
+        setUpOrchestrator()
+        setUpStoreKit2Listener()
+    }
+
+    fileprivate func setUpStoreKit2Listener() {
+        if #available(iOS 15.0, tvOS 15.0, watchOS 8.0, macOS 12.0, *) {
+            orchestrator.storeKit2Listener = MockStoreKit2TransactionListener()
+        }
+    }
+
+    fileprivate func setUpSystemInfo() throws {
+        systemInfo = try MockSystemInfo(platformFlavor: "xyz",
+                                        platformFlavorVersion: "1.2.3",
+                                        finishTransactions: true,
+                                        useStoreKit2IfAvailable: true)
```

Comment:
i quickly just set useStoreKit2IfAvailable to true to run the tests, but will look into the correct way...